src/components/Integrations/DataStudio/FreeAccount/styles/index.css

```diff
@@ -0,0 +1,42 @@
+.check-icon {
```

Comment:
For css we use styled components, like here https://github.com/FromDoppler/doppler-webapp/blob/master/src/components/Reports/GDPRPermissions/SubscriberGdpr.styles.js we have some css but it's not the way we are headed. It's old code. Could you make this into a styled component please?

Reply:
for styled-components with type script i added a dev library in the solution, the css was removed and replace with styles.tsx file with styled componet like to your suggest !!